src/modules/Dropdown/Dropdown.js

```diff
@@ -480,6 +483,15 @@ export default class Dropdown extends Component {
     if (onChange) onChange(e, { ...this.props, value })
   }
 
+  closeOnChange = (e) => {
+    const { closeOnChange, multiple } = this.props
+    const shouldClose = _.isUndefined(closeOnChange)
+      ? !multiple
+      : closeOnChange
```

Comment:
The false branch of this logic is untested, this is why Codecov is failing the PR:

![image](https://cloud.githubusercontent.com/assets/5067638/22437715/fa5d6cde-e6dd-11e6-876d-06fd218b07e6.png)

>**tip**: the coverage data overlayed here in GH is added by the [Codecov extension](https://chrome.google.com/webstore/detail/codecov-extension/keefkhehidemnokodkdkejapdgfjmijf?hl=en-US)

Even though the other branches are executed during tests, we don't actually have assertions for this new feature.  We'll need a new `describe('closeOnChange', ... )` block in the `Dropdown-test.js` where we can test this new prop.